### PR TITLE
ignore .classpath and .project files

### PR DESCRIPTION
**Pull request by housengw**

### CODE REVIEW

File: org.lflang.tests/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Uncomment this line if you wish to ignore the project description file.
+# Typically, this file would be tracked if it contains build/dependency configurations:
+.project
+
+.classpath
```
Comment (lhstrh) [inline org.lflang.tests/.gitignore:5]: `\n` missing

File: org.lflang/.gitignore [hunk 2]
```diff
@@ -1 +1,7 @@
 model/
+
+# Uncomment this line if you wish to ignore the project description file.
+# Typically, this file would be tracked if it contains build/dependency configurations:
+.project
+
+.classpath
```
Comment (lhstrh) [inline org.lflang/.gitignore:7]: `\n` missing

**Comment by lhstrh** [comment]
Looks like this approach won't work. Apparently, you can solve the problem of being suggested to check in changes to these files by locally by doing `git update-index --skip-worktree FILENAME`, but there is no way to configure the repository to have these files but don't track further changes. @edwardalee, @a-sr: what is the reason these files were checked in again? I recall it being something Eclipse/Oomph related...

**Comment by a-sr** [comment]
> @a-sr: what is the reason these files were checked in again? I recall it being something Eclipse/Oomph related...

.project files are required for importing folders as projects into eclipse (during but not limited to the oomph setup).
.classpath, well configures the class path, but these should not be ignored because they are configured specifically for eclipse-plugins and do not represent the default.

**Pull request closed without merging**